client/src/js/samples/components/Edit.js

```diff
@@ -96,6 +107,25 @@ class EditSample extends React.Component {
                             <InputLabel>Locale</InputLabel>
                             <Input name="locale" value={this.state.locale} onChange={this.handleChange} />
                         </InputGroup>
+                        <InputGroup classname="mark-input">
+                            <NotesLabel>Notes</NotesLabel>
+                            <Button onClick={() => this.handlePreview(false)}>Write</Button>
+                            <Button onClick={() => this.handlePreview(true)}>Preview</Button>
+                            {isPreview ? (
+                                <NotesBox>
+                                    <ReleaseMarkdown
+                                        body={notes ? notes : "Edit the sample to add additional notes."}
+                                    ></ReleaseMarkdown>
+                                </NotesBox>
+                            ) : (
+                                <TextArea
+                                    name="notes"
+                                    value={this.state.notes}
+                                    onChange={this.handleChange}
+                                    className="input"
+                                />
+                            )}
```

Comment:
I would suggest moving this to a new component and module, then importing it in this module. This will allow you to keep the handling of the preview-input toggle code separate from the form state in this component.

A new directory should be created at `samples/components/Edit` to store `Edit.js` and the new file: `Notes.js`.